code/__DEFINES/jobs.dm

```diff
@@ -11,6 +33,8 @@
 #define JOB_LOCK_REASON_MAP (1<<1)
 #define JOB_LOCK_REASON_CONFIG (1<<2)
 
+// Job spawn groups
+#define JOB_SPAWN_GROUP_DEPARTMENT "department"
```

Comment:
This define doesn't provide the context well. Is that a group for 'basic jobs of each department'?